awscli/argparser.py

```diff
@@ -16,6 +16,19 @@
 from difflib import get_close_matches
 
 
+HELP_BLURB = (
+    "To see help text, you can run:\n"
+    "\n"
+    "  aws help\n"
+    "  aws <command> help\n"
+    "  aws <command> <subcommand> help\n"
+)
+USAGE = (
+    "aws [options] <command> <subcommand> [parameters]\n"
```

Comment:
Not really. It is actually already an incorrect assumption/help message with the current implementation. Here is how it is currently for wait commands:
```
$ aws ec2 wait

usage: aws [options] <command> <subcommand> [parameters]
aws: error: too few arguments

```
If it was a parameter, the error message would say it is missing a required argument such as:
```
$ aws s3api wait bucket-exists
usage: aws [options] <command> <subcommand> [parameters]
aws: error: argument --bucket is required
```